Gemfile

```diff
@@ -2,4 +2,4 @@ source "https://rubygems.org"
 
 gemspec
 
-gem "activeresource", "~> 4.0.0"
\ No newline at end of file
+gem "activeresource-threadsafe", github: 'Shopify/activeresource'
```

Comment:
Making `shopify_api` *require* our fork of activeresource is a pretty major change.

If we decide to go this way, this probably requires a major version bump of `shopify_api`